.github/workflows/precommit.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             download_name: macos
           - image_name: windows-latest
             download_name: windows
-        python-version: [3.8]
+        python-version: [3.8, 3.9]
```

Comment:
Does this run all tests in both 3.8 and 3.9?

Reply:
Ah, I saw every test has indicated a python version.

Reply:
We don't run udf and doc test for python 3.9. You could find more details in the steps below.